web/themes/custom/pineapple/templates/user/user--account--full.html.twig

```diff
@@ -18,28 +18,87 @@
-    {% include "@components/_hero-image.twig" with {
-      image_url: cover_image_url,
-      hero_attributes: create_attribute().addClass('cover-image')
-    } %}
-    <div class="author-container breakout">
-      <div class="author-info-container container">
-        <div class="profile-image">
-          {% set profile_image_url = bamboo_render_image_style(content.field_u_profile_image|field_target_entity.field_media_image.entity.uri.value, 'thumbnail_115_x_115') %}
-          {% set profile_image_alt = content.field_u_profile_image|field_target_entity.field_media_image.alt %}
-          <img src="{{ profile_image_url }}" alt="{{ profile_image_alt }}">
+    <div class="sidebar-container">
+      <div class="author-container">
+        <div class="cover-image-container">
+          {% set cover_image_url = bamboo_render_image_style(content.field_u_cover_image|field_target_entity.field_media_image.entity.uri.value, 'vertical_teaser') %}
+          {% set cover_image_alt = content.field_u_cover_image|field_target_entity.field_media_image.alt %}
+          <img src="{{ cover_image_url }}" alt="{{ cover_image_alt }}">
         </div>
-        <div class="author-text">
+        <div class="author-name container">
+          <div class="profile-image">
+            {% set profile_image_url = bamboo_render_image_style(content.field_u_profile_image|field_target_entity.field_media_image.entity.uri.value, 'thumbnail_115_x_115') %}
+            {% set profile_image_alt = content.field_u_profile_image|field_target_entity.field_media_image.alt %}
+            <img src="{{ profile_image_url }}" alt="{{ profile_image_alt }}">
+          </div>
           {% include "@components/_heading.twig" with {
             heading_level: 1,
-            heading_attributes: create_attribute().addClass('author-name'),
+            heading_attributes: create_attribute().addClass('subheading'),
             heading_text: content.field_u_name|field_value|first['#context'].value
           } %}
+        </div> <!-- End of author-name-container -->
+        <div class="author-info">
+          {% set current_user = bamboo_load_currentuser().uid.value %}
+          {% set user_roles = user.getRoles() %}
+          {% set member_since = user.getCreatedTime()|date("m/d/Y") %}
+          <div class="member-since">
+            <strong>Member since</strong>
+            <p>{{ member_since }}</p>
+          </div>
+          <div class="roles">
+            {% if 'recipe_editor' in user_roles or 'article_editor' in user_roles %}
+              <strong>Role(s)</strong>
+            {% endif %}
+            {% if 'recipe_editor' in user_roles %}
+              <p>Recipe Editor</p>
+            {% endif %}
+            {% if 'article_editor' in user_roles %}
+              <p>Article Editor</p>
+            {% endif %}
+          </div> <!-- End of roles -->
```

Comment:
If a use is just authenticated, the only see the roles label. Update this so that they either don't see the label or it just says authenticated.

Reply:
this has been updated already.

Reply:
Check updated comment.

Reply:
done